Makefile

```diff
@@ -37,6 +37,13 @@ install:
 	# Copy dracut and systemd config files
 	cp -Prf baseimage/dracut $(DESTDIR)$(prefix)/share/doc/bootc/baseimage/dracut
 	cp -Prf baseimage/systemd $(DESTDIR)$(prefix)/share/doc/bootc/baseimage/systemd
+	# Install bootc-destructive-cleanup in fedora derivatives 
+	@ID=$$(grep '^ID=' /usr/lib/os-release | cut -d'=' -f2 | tr -d '"'); \
+	ID_LIKE=$$(grep '^ID_LIKE=' /usr/lib/os-release | cut -d'=' -f2 | tr -d '"'); \
+	if [ "$$ID" = "fedora" ] || [ "$$ID_LIKE" = "fedora" ]; then \
```

Comment:
`ID_LIKE` needs to be a substring match
```
$ podman run --rm quay.io/centos-bootc/centos-bootc:stream10 grep ID_LIKE /usr/lib/os-release
ID_LIKE="rhel fedora"
$
```

Maybe we can factor out a `lib/install.sh` or something?